src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1540,10 +1543,10 @@ public override BoundNode VisitNullCoalescingOperator(BoundNullCoalescingOperato
             }
 
             bool? resultIsNullable = getIsNullable(leftOperand, leftResult) == false ? false : getIsNullable(rightOperand, rightResult);
-            _result = TypeSymbolWithAnnotations.Create(resultType, resultIsNullable);
+            _resultType = TypeSymbolWithAnnotations.Create(resultType, resultIsNullable);
             return null;
 
-            bool? getIsNullable(BoundExpression e, Result r) => (r.Type is null) ? e.IsNullable() : r.Type.IsNullable;
+            bool? getIsNullable(BoundExpression e, TypeSymbolWithAnnotations t) => (t is null) ? e.IsNullable() : t.IsNullable;
```

Comment:
Consider adding a newline between local function defs for readability, especially the one-line one.